binary_transparency/firmware/devices/usb_armory/bootloader/led.go

```diff
@@ -0,0 +1,43 @@
+//+build armory
+
+package main
+
+import (
+	"fmt"
+	"time"
+
+	usbarmory "github.com/f-secure-foundry/tamago/board/f-secure/usbarmory/mark-two"
+)
+
+// blinkOnOff blinks the specified LED on and then off.
+func blinkOnOff(led string, on, off time.Duration) {
+	usbarmory.LED(led, true)
+	<-time.After(on)
+	usbarmory.LED(led, false)
+	<-time.After(off)
+}
+
+// haltAndCatchFire signals for help and never returns.
+func haltAndCatchFire(msg string, code int) {
+	fmt.Println(msg)
```

Comment:
Maybe it doesn't matter, but I think this will go to stdout and I would expect stderr

Reply:
I suspect it doesn't really matter here - we're on a tty on the end of a serial console at this point.